Cubical/Structures/Axioms.agda

```diff
@@ -0,0 +1,45 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.Structures.Axioms where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.Path
+open import Cubical.Foundations.SIP
+open import Cubical.Data.Sigma
+
+private
+  variable
+    ℓ ℓ₁ ℓ₁' ℓ₂ : Level
+
+-- Now, we want to add axioms (i.e. propositions) to our Structure S that don't affect the ι.
```

Comment:
Can you update this comment to something suitable? Maybe move it to the header of the file as well